### ISSUE TITLE
Link on github front page is malicious

### ISSUE THREAD

**Issue by MarkFullerInflxr** [gregnb/mui-datatables#2023]
When I follow the link 

https://www.material-ui-datatables.com/

My computer warns me of suspicious activity and viruses. If I use a web tool to analyze it, it agrees. 

## Expected Behavior
not malware

## Current Behavior
malware

## Steps to Reproduce (for bugs)
1. click the link from the front page of the repo
2. malware!

**Comment by BThero** [comment]
Yeah this was already reported in #2003. Doesn't seem like the fix is coming soon though...
